modules/intro/pages/continuous-availability-overview.adoc

```diff
@@ -7,14 +7,14 @@ HA (High Availability) is a generic term used to describe a computer system whic
 With Continuous Availability, TigerGraph goes beyond the standard scope and definition of High Availability by providing the following functionality:
 
 * Fault tolerance against loss of database server(s)
- ** Automated recovery of services for Intra-cluster failure
+ ** Automated recovery of services in case of intra-cluster failure
 * Full native HA support for user-facing applications - Seamless automatic client reconnection to standby GSQL server and GraphStudio servers
-* Failover to remote cluster during Disaster Recovery
-* Improve RoI with additional Replicas
- ** Enhanced Query Throughput performance
+* Failover to remote cluster during disaster decovery
```

Comment:
1. typo: decovery -> recovery
2. This comparison chart is not just informational, but it is Promotional material, where terms are key terms are highlighted. In this case, they are industry-standard terms help the reader to see what features are supported. I think you should have left it the way it was.